prow/plugins/milestonestatus/milestonestatus_test.go

```diff
@@ -116,7 +126,13 @@ func TestMilestoneStatus(t *testing.T) {
 			User:   github.User{Login: tc.commenter},
 		}
 
-		if err := handle(fakeClient, logrus.WithField("plugin", pluginName), e, 42); err != nil {
+		repoMilestone := map[string]plugins.Milestone{"": {MaintainersID: 42}}
+
+		if !tc.noRepoMaintainer {
+			repoMilestone["org/repo"] = plugins.Milestone{MaintainersID: 42}
```

Comment:
same comment re: not knowing whether the plugin is pulling the right value from the right place